pulp_rpm/app/urls.py

```diff
@@ -3,7 +3,11 @@
 
 from .viewsets import CopyViewSet, CompsXmlViewSet
 
-V3_API_ROOT = settings.V3_API_ROOT_NO_FRONT_SLASH
+
+if hasattr(settings, "V3_API_ROOT_NO_FRONT_SLASH"):
+    V3_API_ROOT = settings.V3_API_ROOT_NO_FRONT_SLASH
+else:
+    V3_API_ROOT = "/pulp/api/v3/"
```

Comment:
👍🏽

Reply:
Note: I just changed this, I think the path fragment needs to not have a leading slash